README.md

```diff
@@ -34,15 +34,23 @@ and after a few moments everything should be up and ready. The script will creat
 Install:
 
 ``` bash
-dnf install -y podman podman-plugins podman-compose
+dnf install -y podman podman-plugins podman-compose podman-docker
+```
+
+Enable socket environment:
```

Comment:
The whole paragraph is missing context - why you need to enable socket, is this needed for running in podman or just compiling the project etc. (I know the answers, but please explain things in the README).

Reply:
I added a note... The README is too long as is.

It will help - a bit - once we have a newer npm environment that supports OpenSSL 1.1.x and 3 out of the box, and doesn't need the hack